INSTALLATION.md

```
@@ -623,8 +673,10 @@ Restart the Codex CLI or reload your IDE extension to load the MCP server.
 
 Ask Codex:
 ```
+
 Check the health of the AutoMem service
-```
+
+````
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)